pages/dkp/kaptain/2.0.0/install/konvoy-dkp/index.md

```diff
@@ -0,0 +1,195 @@
+
+- When installing on Konvoy 1.x, ensure the following Kubernetes base addons that are needed by Kaptain are enabled:
+
+  ```yaml
+  - configRepository: https://github.com/mesosphere/kubernetes-base-addons
+    configVersion: stable-1.20-4.1.0
+    addonsList:
+      - name: istio
+        enabled: true
+      - name: dex
+        enabled: true
+      - name: cert-manager
+        enabled: true
+      - name: prometheus
+        enabled: true
+  ```
+
+- Add the Kaptain addon repository to your Konvoy `cluster.yaml` to install Kaptain dependencies:
+  ```yaml
+  - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
+    configVersion: stable-1.20-1.4.0
+    addonsList:
+      - name: knative
+        enabled: true
+  ```
+- For GPU deployment, follow the instructions in [Konvoy GPU documentation][konvoy-gpu].
+- Then follow the [Konvoy documentation][konvoy_deploy_addons] to deploy the addons.
+
+## Prerequisites for DKP 2.x
+
+For DKP 2.x, ensure the following applications are enabled in Kommander:
+
+- Use the existing Kommander configuration file, or initialize the default one:
+  ```
+  kommander install --init > kommander-config.yaml
+  ```
+- Ensure the following applications are enabled in the config:
+  ```yaml
+  apiVersion: config.kommander.mesosphere.io/v1alpha1
+  kind: Installation
+  apps:
+    ...
+    dex:
+    dex-k8s-authenticator:
+    kube-prometheus-stack:
+    istio:
+    knative:
+    minio-operator:
+    traefik:
+    nvidia:  # to enable GPU support
+    ...
+  ```
+- For GPU deployment, follow the instructions in [Kommander GPU documentation][kommander-gpu].
+
+- Apply the new configuration to Kommander:
+  ```
+  kommander install --installer-config kommander-config.yaml
+  ```
+  Check [Kommander installation documentation][kommander-install] for more information.
+
+<p class="message--note"><strong>NOTE: </strong>Starting from the 1.3 release, Spark Operator is no longer installed by default with Kaptain.</p>
+
+In case you need to run Spark jobs on Kubernetes using Spark Operator, it needs to be installed separately.
+Use the following instructions to install Spark Operator from Kommander Catalog for your target platform:
+[Konvoy 1.x][install-spark-konvoy1] or [DKP 2.x][install-spark-dkp2]
+
+## Install Kaptain
+
+- Install the [kubectl-kudo CLI plugin][kudo_cli]
+
+- After the Konvoy cluster has been deployed (including Istio and KNative), install KUDO:
+
+  ```bash
+  kubectl kudo init --wait
+  ```
+
+- Download [kubeflow-1.4.0_1.3.0.tgz][download] tarball.
+<p class="message--note"><strong>NOTE: </strong>Starting with Kaptain 1.2.0, automatic profile creation on initial login is now disabled by default. See <a href="../../user-management">User Management</a> for more details.</p>
+
+- Set required configuration based on the target platform:
+  - When installing on Konvoy 1.x, add the following configuration to `parameters.yaml` file:
+  ```bash
+  cat >> parameters.yaml << END
```

Comment:
seems like this line messed with github code highlighting... possibly the reason why vale crashes?